test/snapshots/index.js.md

```diff
@@ -55,12 +77,41 @@ Generated by [AVA](https://ava.li).
     var a = baz.a,␊
         rest = _objectWithoutPropertiesLoose(baz, ["a"]);`
 
+## inlines React elements by default in production environment
+
+> Snapshot 1
+
+    `var REACT_ELEMENT_TYPE;␊
+    ␊
+    function _jsx(type, props, key, children) { if (!REACT_ELEMENT_TYPE) { REACT_ELEMENT_TYPE = typeof Symbol === "function" && Symbol.for && Symbol.for("react.element") || 0xeac7; } var defaultProps = type && type.defaultProps; var childrenLength = arguments.length - 3; if (!props && childrenLength !== 0) { props = { children: void 0 }; } if (props && defaultProps) { for (var propName in defaultProps) { if (props[propName] === void 0) { props[propName] = defaultProps[propName]; } } } else if (!props) { props = defaultProps || {}; } if (childrenLength === 1) { props.children = children; } else if (childrenLength > 1) { var childArray = new Array(childrenLength); for (var i = 0; i < childrenLength; i++) { childArray[i] = arguments[i + 3]; } props.children = childArray; } return { $$typeof: REACT_ELEMENT_TYPE, type: type, key: key === undefined ? null : '' + key, ref: null, props: props, _owner: null }; }␊
+    ␊
+    export default function MyComponent(props) {␊
+      return _jsx("div", {␊
+        height: 20␊
+      }, "abc");␊
+    }`
+
+## inlines React elements with external helper when {"runtime":true} in production environment
+
+> Snapshot 1
+
+    `import _jsx from "@babel/runtime/helpers/esm/jsx";␊
+    export default function MyComponent(props) {␊
+      return _jsx("div", {␊
+        height: 20␊
+      }, "abc");␊
+    }`
+
 ## removes prop types by default in production environment
 
 > Snapshot 1
 
-    `export default function MyComponent(props) {␊
-      return React.createElement("div", null);␊
+    `var REACT_ELEMENT_TYPE;␊
```

Comment:
Do we care that this test is modified as well? Doesn't seem like an issue but if we do then we should add `{inlineReactElements: false}` to the `options`